RELEASE.md

```diff
@@ -107,6 +107,7 @@ To publish a release candidate:
 1. Wait until the CI pipeline succeeds (once a tag is created, the release process through GitHub Actions will be triggered for this tag).
 1. Merge the release branch `release-x.y` into `main` (see [Merging release branch into main](#merging-release-branch-into-main))
 1. Create a pre-release on GitHub. See [Creating release on GitHub](#creating-release-on-github).
+1. Optionally create [Helm release](#helm-release).
```

Comment:
Any reason not to do it? I would leave less decision making to the person following the process.

Reply:
or rather I should've said - make the decision making easier for the person. What are the cases when we don't want to do to a release?

Reply:
true, fixing
```suggestion
1. Create [Helm release](#helm-release).
```